website_medical_prescription_order_line/views/medical_prescription_order_line_template.xml

```diff
@@ -19,83 +18,60 @@
                 If you would like to order a medication, please click the green
                 button to add to cart.
             </h5>
-            <table class="table table-hover table-responsive">
-                <thead>
-                  <tr class="active">
-                      <th colspan="3">Patient</th>
-                      <th colspan="2">Rx Number</th>
-                      <th colspan="3">Medication</th>
-                      <!--<th colspan="1">Written</th>-->
-                      <th colspan="1">Remain</th>
-                      <th colspan="2">Last Disp</th>
-                      <th colspan="2">Expiration</th>
-                      <th colspan="1" class="text-right">Order</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    <t t-foreach="prescription_order_lines" t-as="line">
-                        <tr>
-                            <td colspan="3">
-                                <span t-field="line.patient_id.name" />
-                            </td>
-                            <td colspan="2" class="text-nowrap">
-                                <span t-field="line.name" />
-                            </td>
-                            <td colspan="3">
-                                <span t-field="line.medicament_id.display_name" />
-                            </td>
-                            <!--<td colspan="1">
-                                <span t-field="line.qty" />
-                            </td>-->
-                            <td colspan="1">
-                                <span t-field="line.can_dispense_qty" />
-                            </td>
-                            <td colspan="2">
+            <div class="container">
+                <t t-foreach="prescription_order_lines" t-as="line">
+                    <div class="col-xs-12 col-sm-6">
+                        <div class="panel panel-default">
+                            <div class="panel-heading">
+                                <span t-field="line.patient_id.name" /> - <span t-field="line.medicament_id.display_name" />
+                            </div>
+                            <div class="panel-body">
+                                <label class="label label-info">Rx Number</label>
+                                <span t-field="line.name" /><br/>
```

Comment:
General rule - if you're using a `<br/>`, you're probably doing it wrong. Similar to tables, really. There's a time & a place, but for the most part we design around them.

In this case, I believe encapsulating the label and spans in `<p>` would a slightly better spacing too.
```
<p>
    <label />
    <span />
</p>
```